### PR TITLE
Update rnn_cell.py

### PR DESCRIPTION
**Pull request by gautam1858**
converting to list of RNNCells

## Description ##
(Brief description on what this PR is about)

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [ ] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [ ] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

**Comment by piiswrong** [comment]
@szha

File: python/mxnet/rnn/rnn_cell.py [hunk 1]
```diff
@@ -39,11 +39,13 @@ def _cells_begin_state(cells, **kwargs):
     return sum([c.begin_state(**kwargs) for c in cells], [])
 
 def _cells_unpack_weights(cells, args):
+    cells = [cells]
```
Comment (szha) [inline python/mxnet/rnn/rnn_cell.py]: why? is this a bug? is there a test case that triggers this bug?

**Comment by gautam1858** [comment]
yup its a bug

**Comment by szha** [comment]
how is it triggered?

**Comment by piiswrong** [comment]
This looks very suspicious. even if there is a bug I don't think this is the right way to fix it

**Pull request closed without merging**